addon/mixins/in-viewport.js

```diff
@@ -59,7 +59,9 @@ export default Ember.Mixin.create({
   },
 
   _setupElement: on('didInsertElement', function() {
-    if (!canUseDOM) {
+    const noSetup = !get(this, 'viewportSpy') && get(this, 'noSetup');
```

Comment:
I would suggest ignoring `viewportSpy` here. When someone wants to disable the component, they shouldn't have to worry about the other option values.